packages/components/src/mobile/bottom-sheet/bottom-sheet-navigation/navigation-screen.native.js

```diff
@@ -69,10 +77,28 @@ const BottomSheetNavigationScreen = ( { children, fullScreen } ) => {
 			setHeightDebounce( height );
 		}
 	};
-
 	return useMemo( () => {
-		return <View onLayout={ onLayout }>{ children }</View>;
-	}, [ children, isFocused ] );
+		return isScrollable || isNested ? (
+			<View onLayout={ onLayout }>{ children }</View>
+		) : (
+			<ScrollView { ...listProps }>
+				<TouchableHighlight accessible={ false }>
+					<View onLayout={ onLayout }>
+						{ children }
+						{ ! isNested && (
+							<View
+								style={ {
+									height:
+										safeAreaBottomInset ||
+										styles.scrollableContent.paddingBottom,
```

Comment:
The same [idea](https://github.com/WordPress/gutenberg/pull/26204/files#r516549256) we can apply there